appveyor.yml

```diff
@@ -20,7 +20,7 @@ environment:
   NOT_CRAN: true
   KEEP_VIGNETTES: true
   R_ARCH: x64
-  R_VERSION: "4.0.5"
+  R_VERSION: "4.1.0"

```

Comment:
@Yuri05 is 4.1 supported by the R validated environment?

Reply:
oh wait, it's appveyor, not the minimal package version.
When compiled for 4.0, the package still runs under 3.6
But if we compile under 4.1 - it must be tested if it runs under 3.6

Reply:
LOL

We might as well roll back to [S language](https://en.wikipedia.org/wiki/S_(programming_language)). 🤣

Reply:
LMAO